### PR TITLE
added helm chart

### PR DESCRIPTION
**Pull request by yaron2**
Takes care of #39 and should also address #11

### CODE REVIEW

File: helm-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        - name: AZURE_CLIENT_ID
+          value: {{ .Values.env.azureclientid }}
+        - name: AZURE_CLIENT_KEY
+          value: {{ .Values.env.azureclientkey }}
+        - name: AZURE_TENANT_ID
+          value: {{ .Values.env.azuretenantid }}
+        - name: AZURE_SUBSCRIPTION_ID
+          value: {{ .Values.env.azuresubscriptionid }}
+        - name: ACI_RESOURCE_GROUP
+          value: {{ .Values.env.aciresourcegroup }}
```
Comment (lachie83) [inline helm-chart/templates/deployment.yaml]: Nit: Can you add newlines at the end of all files

File: helm-chart/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: aci-connector
+version: 1.0.0
```
Comment (lachie83) [inline helm-chart/Chart.yaml]: I would personally set this to 0.1.0 to indicate the level of maturity of the chart.

File: helm-chart/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: aci-connector
```
Comment (lachie83) [inline helm-chart/Chart.yaml]: The directory name must match the chart name to make the linter pass. I recommend the following charts/aci-connector

Comment (lachie83) [inline helm-chart/Chart.yaml]: Can you please update the docs with an optional section to use the helm chart for installation

**Comment by yaron2** [comment]
@lachie83 ready for review

File: README.md [hunk 4]
```diff
@@ -114,6 +114,12 @@ k8s-agentpool1-31868821-2   Ready                      5d        v1.7.0
 k8s-master-31868821-0       Ready,SchedulingDisabled   5d        v1.7.0
 ```
 
+### Install the ACI Connector with Helm
```
Comment (lachie83) [inline README.md]: Can you put (optional)

File: charts/aci-connector/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: AZURE_CLIENT_ID
```
Comment (lachie83) [inline charts/aci-connector/templates/deployment.yaml]: Can you add ACI_REGION and default to westus -- https://github.com/Azure/aci-connector-k8s/blob/master/connector.ts#L44-L46

File: charts/aci-connector/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+name: aci-connector
+version: 0.1.0
+description: a Helm chart to install aci-connector inside a Kubernetes cluster.
+sources:
+  - https://github.com/Azure/aci-connector-k8s
+maintainers:
+  - name: Yaron Schnedier
+    email: yaronsc@microsoft.com
+
```
Comment (lachie83) [inline charts/aci-connector/Chart.yaml]: Nit: no tab on newline

File: README.md [hunk 7]
```diff
@@ -114,6 +114,12 @@ k8s-agentpool1-31868821-2   Ready                      5d        v1.7.0
 k8s-master-31868821-0       Ready,SchedulingDisabled   5d        v1.7.0
 ```
 
+### Install the ACI Connector with Helm (optional)
+
+```console
+$ helm install --name my-release ./charts/aci-connector
```
Comment (lachie83) [inline README.md]: Please also provide details on either updating the values.yaml or running via the --set option

File: charts/aci-connector/values.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+image:
+  repository: microsoft/aci-connector-k8s
+  tag: latest
+  pullPolicy: Always
+env:
+  azureclientid:
```
Comment (lachie83) [inline charts/aci-connector/values.yaml]: Please consistently use lowerCamelCase like you have for line 4

**Comment by brendandburns** [comment]
Many thanks. Once @lachie83 is good with this, I'm excited to see it merge.

File: charts/aci-connector/templates/deployment.yaml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+          value: {{ .Values.env.azureclientkey }}
+        - name: AZURE_TENANT_ID
+          value: {{ .Values.env.azuretenantid }}
+        - name: AZURE_SUBSCRIPTION_ID
+          value: {{ .Values.env.azuresubscriptionid }}
+        - name: ACI_RESOURCE_GROUP
+          value: {{ .Values.env.aciresourcegroup }}
+        - name: ACI_REGION
+          value: {{ default "westus" .Values.env.aciregion }}
+
```
Comment (lachie83) [inline charts/aci-connector/templates/deployment.yaml]: Looks like you still have indents on your newline on most of the files. Can you fix?

**Comment by lachie83** [comment]
Thanks @yaron2. I've made a few changes. LGTM

**Pull request merged**